index.js

```diff
@@ -44,6 +26,6 @@ export default (config = {}) => {
     forex: Forex(config),
     crypto: Crypto(config),
     technical: Technical(config),
-    performance: Performance(config)
+    performance: Performance(config),
```

Comment:
Please remove this trailing comma, the lint script will do this: `npm run lint`

Reply:
Prettier updated and has this behavior by default now - I'll add the option to turn off this feature.

Reply:
Wait, I'm wrong - the lint write rules don't target these files. I extracted the files into a .prettierrc so this shouldn't happen again 😄